.ci.yaml

```diff
@@ -60,6 +62,88 @@ targets:
         ]
     scheduler: luci
 
+  - name: windows-build+platform-tests master
+    recipe: plugins/plugins
+    bringup: true
+    timeout: 30
+    properties:
+      # TODO(stuartmorgan): Uncomment when removing bringup.
+      #add_recipes_cq: "true"
```

Comment:
Is this necessary? My expectation is this could land in this PR

Reply:
Probably not for this particular one since it's a copy-paste, but in general I would rather make sure a new target actually works in the wild before adding it to the recipe CQ since I really don't want to bring the recipes repo to a standstill by throwing untested things into its pipeline for landing changes.

Reply:
This is a really good insight! I'll have to add it to the documentation (and later as a test)